client/js/shout.js

```diff
@@ -382,6 +382,28 @@ $(function() {
 		}
 	}
 
+	if ($.cookie("timestamp")) {
+		$("#timestamp").val($.cookie("timestamp")).change();
+	} else {
+		$("#timestamp").val("HH:mm").change();
+	}
+
+	$("#timestamp").on("change", function() {
+		var timestamp = $(this).val();
+		Handlebars.unregisterHelper("tz");
```

Comment:
Well, there are a couple of issues with that:

1. I don't like global variables, they get quite messy. If the handler was in a module that we passed the variable to, then I'd be all for that, but the way it works, that's not great.
2. The module that the handler is in currently minified, which isn't a big issue, but does mean I'd need to set it to window.timestamp rather than just timestamp as well.

Basically, with the way we currently have the code, I think this is probably the least of the evils.